src/toast/styled-components.js

```diff
@@ -147,7 +147,7 @@ export const CloseIconSvg = styled<SharedStylePropsArgT>('svg', props => {
   return {
     ...getSvgStyles(props),
     cursor: 'pointer',
-    width: '16px',
+    width: props.$size || '16px',
```

Comment:
what's this for?

Reply:
lol I spent maybe 2 hours investigating why when I overridded the `$size` property on `CloseIcon` in `Toast` it was always clamped to `16px`. Turned out that it was hard-coded here